packages/runtime/src/lib/parsing.ts

```diff
@@ -282,4 +283,19 @@ export function bigEndianBytesToBigInt(bytes: Uint8Array | Buffer): bigint {
  */
 export function strHexToBytes(str: string): Uint8Array {
 	return new Uint8Array(Buffer.from(str.slice(2), "hex"));
+}
+
+/**
+ * assert if string given is a valid JSON object
+ * @param jsonString
+ */
+ export function assertJSON(jsonString: string, line: number): void {
+	const strictBigJSON = JSONbig({ strict: true });
+	try {
+		strictBigJSON.parse(jsonString);
+	} catch (e) {
```

Comment:
do we catch all the error here?

Reply:
Yes we catch all the errors from JSON.parse

Reply:
Should we handle them? To give a better feedback to the user when it throws an error

Reply:
Well this one function `assertJSON` just to make sure the bytes we parse is correct JSON format. Incase the string was wrong, we throw error invalid json parsing